iftop fails to build on amd64

The package iftop with version 0.17 and epoch 41 fails to build from source on amd64 with the following error:

```
2025-06-24 07:21:40.741332834 +0000 UTC parsed env
2025-06-24 07:21:40.741562448 +0000 UTC configuring puller identity "720909c9f5279097d847ad02a2f24ba8f59de36a/a49c7fedc33adf69"...
2025-06-24 07:21:40.742078997 +0000 UTC running command chainctl [auth login --audience apk.cgr.dev --identity 720909c9f5279097d847ad02a2f24ba8f59de36a/a49c7fedc33adf69]
2025-06-24 07:21:41.179636663 +0000 UTC Successfully exchanged token.
2025-06-24 07:21:41.29721042 +0000 UTC Valid! Id: 720909c9f5279097d847ad02a2f24ba8f59de36a/a49c7fedc33adf69
2025-06-24 07:21:41.36497995 +0000 UTC Updates are available for chainctl (current version: 0.2.91; latest: 0.2.92). To install, please run:
2025-06-24 07:21:41.364983699 +0000 UTC     $ chainctl update
2025-06-24 07:21:41.375838344 +0000 UTC command "chainctl" completed successfully
2025-06-24 07:21:41.375933253 +0000 UTC puller identity configured successfully
2025-06-24 07:21:41.375946173 +0000 UTC puller identity configured successfully
2025-06-24 07:21:41.375956337 +0000 UTC running build...
2025-06-24 07:21:41.37596607 +0000 UTC running command melange [build iftop.yaml --gcplog --source-dir iftop --arch x86_64 --arch=x86_64 --env-file=build-x86_64.env --namespace=wolfi --pipeline-dir=pipelines --out-dir=./packages/ --runner=qemu --keyring-append=https://packages.wolfi.dev/os/wolfi-signing.rsa.pub --repository-append=https://packages.wolfi.dev/os --repository-append=https://apk.cgr.dev/wolfi-presubmit/world-1750746723102 --repository-append=https://apk.cgr.dev/wolfi-presubmit/world-1750746723102 --git-repo-url=https://github.com/wolfi-dev/os]
2025-06-24 07:21:41.416580093 +0000 UTC melange v0.26.12+dirty is building:
2025-06-24 07:21:41.434708994 +0000 UTC populating workspace /tmp/melange-workspace-855737706 from iftop
2025-06-24 07:21:41.573899878 +0000 UTC Updates are available for chainctl (current version: 0.2.91; latest: 0.2.92). To install, please run:
2025-06-24 07:21:41.573904158 +0000 UTC     $ chainctl update
2025-06-24 07:21:43.753772403 +0000 UTC image configuration:
2025-06-24 07:21:43.754064005 +0000 UTC   contents:
2025-06-24 07:21:43.754099038 +0000 UTC     build repositories: [https://apk.cgr.dev/wolfi-presubmit/world-1750746723102 https://packages.wolfi.dev/os]
2025-06-24 07:21:43.75412529 +0000 UTC     runtime repositories: []
2025-06-24 07:21:43.754145483 +0000 UTC     keyring:      [https://packages.wolfi.dev/os/wolfi-signing.rsa.pub]
2025-06-24 07:21:43.754162005 +0000 UTC     packages:     [autoconf=2.72-r3 automake=1.18-r1 binutils=2.44-r9 build-base=1-r8 busybox=1.37.0-r46 ca-certificates-bundle=20241121-r42 gcc-14-default=14.3.0-r2 gcc-14=14.3.0-r2 gfortran-14=14.3.0-r2 glibc-dev=2.41-r50 glibc-locale-posix=2.41-r50 glibc=2.41-r50 gmp=6.3.0-r7 isl=0.27-r3 ld-linux=2.41-r50 libbz2-1=1.0.8-r18 libcrypt1=2.41-r50 libcrypto3=3.5.0-r3 libgcc=15.1.0-r1 libgfortran-14=14.3.0-r2 libgfortran=15.1.0-r1 libpcap-dev=1.10.5-r2 libpcap=1.10.5-r2 libquadmath=15.1.0-r1 libssl3=3.5.0-r3 libstdc++-14-dev=14.3.0-r2 libstdc++-14=14.3.0-r2 libstdc++=15.1.0-r1 libxcrypt-dev=4.4.38-r2 libxcrypt=4.4.38-r2 libzstd1=1.5.7-r2 linux-headers=6.15.3-r0 m4=1.4.20-r4 make=4.4.1-r7 mpc=1.3.1-r6 mpfr=4.2.2-r1 ncurses-dev=6.5_p20241228-r3 ncurses-terminfo-base=6.5_p20241228-r3 ncurses=6.5_p20241228-r3 nss-db=2.41-r50 nss-hesiod=2.41-r50 openssf-compiler-options=20240627-r21 perl=5.40.2-r1 pkgconf=2.5.0-r0 posix-cc-wrappers=1-r6 scanelf=1.3.8-r4 wget=1.25.0-r2 wolfi-baselayout=20230201-r21 zlib=1.3.1-r50]
2025-06-24 07:21:43.754202972 +0000 UTC   accounts:
2025-06-24 07:21:43.754220248 +0000 UTC     runas:  
2025-06-24 07:21:43.754237431 +0000 UTC     users:
2025-06-24 07:21:43.754255598 +0000 UTC       - uid=1000(build) gid=1000
2025-06-24 07:21:43.754281781 +0000 UTC     groups:
2025-06-24 07:21:43.754300699 +0000 UTC       - gid=1000(build) members=[build]
2025-06-24 07:21:43.75432967 +0000 UTC auth configured for: []
2025-06-24 07:21:43.817433822 +0000 UTC installing wolfi-baselayout (20230201-r21)
2025-06-24 07:21:43.91266063 +0000 UTC installing ca-certificates-bundle (20241121-r42)
2025-06-24 07:21:43.91341946 +0000 UTC installing glibc-locale-posix (2.41-r50)
2025-06-24 07:21:43.914326488 +0000 UTC installing libgcc (15.1.0-r1)
2025-06-24 07:21:44.807244292 +0000 UTC installing glibc (2.41-r50)
2025-06-24 07:21:44.87653241 +0000 UTC installing ld-linux (2.41-r50)
2025-06-24 07:21:44.877502135 +0000 UTC installing libbz2-1 (1.0.8-r18)
2025-06-24 07:21:44.878038342 +0000 UTC installing libxcrypt (4.4.38-r2)
2025-06-24 07:21:44.880439213 +0000 UTC installing libcrypt1 (2.41-r50)
2025-06-24 07:21:44.88149423 +0000 UTC installing zlib (1.3.1-r50)
2025-06-24 07:21:46.388213877 +0000 UTC installing perl (5.40.2-r1)
2025-06-24 07:21:46.396673473 +0000 UTC installing m4 (1.4.20-r4)
2025-06-24 07:21:46.397653823 +0000 UTC installing autoconf (2.72-r3)
2025-06-24 07:21:46.398380792 +0000 UTC installing automake (1.18-r1)
2025-06-24 07:21:46.399243369 +0000 UTC installing libstdc++ (15.1.0-r1)
2025-06-24 07:21:46.399906436 +0000 UTC installing libzstd1 (1.5.7-r2)
2025-06-24 07:21:47.492164345 +0000 UTC installing binutils (2.44-r9)
2025-06-24 07:21:47.494938543 +0000 UTC installing libxcrypt-dev (4.4.38-r2)
2025-06-24 07:21:47.495789122 +0000 UTC installing nss-db (2.41-r50)
2025-06-24 07:21:47.49722524 +0000 UTC installing nss-hesiod (2.41-r50)
2025-06-24 07:21:47.499852623 +0000 UTC installing linux-headers (6.15.3-r0)
2025-06-24 07:21:47.510150146 +0000 UTC installing glibc-dev (2.41-r50)
2025-06-24 07:21:47.513268213 +0000 UTC installing libquadmath (15.1.0-r1)
2025-06-24 07:21:47.517429527 +0000 UTC installing libstdc++-14 (14.3.0-r2)
2025-06-24 07:21:47.51813664 +0000 UTC installing libstdc++-14-dev (14.3.0-r2)
2025-06-24 07:21:47.526270666 +0000 UTC installing posix-cc-wrappers (1-r6)
2025-06-24 07:21:47.528933956 +0000 UTC installing gmp (6.3.0-r7)
2025-06-24 07:21:47.529969603 +0000 UTC installing isl (0.27-r3)
2025-06-24 07:21:47.531149515 +0000 UTC installing mpfr (4.2.2-r1)
2025-06-24 07:21:47.532589767 +0000 UTC installing mpc (1.3.1-r6)
2025-06-24 07:21:47.534006113 +0000 UTC installing openssf-compiler-options (20240627-r21)
2025-06-24 07:21:51.26071888 +0000 UTC installing gcc-14 (14.3.0-r2)
2025-06-24 07:21:51.263412997 +0000 UTC installing libgfortran-14 (14.3.0-r2)
2025-06-24 07:21:51.263978813 +0000 UTC installing gfortran-14 (14.3.0-r2)
2025-06-24 07:21:51.264641627 +0000 UTC installing libgfortran (15.1.0-r1)
2025-06-24 07:21:51.265129547 +0000 UTC installing gcc-14-default (14.3.0-r2)
2025-06-24 07:21:51.265665208 +0000 UTC installing make (4.4.1-r7)
2025-06-24 07:21:51.266315307 +0000 UTC installing pkgconf (2.5.0-r0)
2025-06-24 07:21:51.266856246 +0000 UTC installing build-base (1-r8)
2025-06-24 07:21:51.267086011 +0000 UTC installing busybox (1.37.0-r46)
2025-06-24 07:21:51.267544839 +0000 UTC installing libcrypto3 (3.5.0-r3)
2025-06-24 07:21:51.268023084 +0000 UTC installing libpcap (1.10.5-r2)
2025-06-24 07:21:51.268395546 +0000 UTC installing libpcap-dev (1.10.5-r2)
2025-06-24 07:21:51.268923006 +0000 UTC installing libssl3 (3.5.0-r3)
2025-06-24 07:21:51.269627501 +0000 UTC installing ncurses-terminfo-base (6.5_p20241228-r3)
2025-06-24 07:21:51.270881459 +0000 UTC installing ncurses (6.5_p20241228-r3)
2025-06-24 07:21:51.271400575 +0000 UTC installing ncurses-dev (6.5_p20241228-r3)
2025-06-24 07:21:51.272020457 +0000 UTC installing scanelf (1.3.8-r4)
2025-06-24 07:21:51.272435528 +0000 UTC installing wget (1.25.0-r2)
2025-06-24 07:21:59.407360098 +0000 UTC qemu: generating ssh key pairs for ephemeral VM
2025-06-24 07:21:59.409402522 +0000 UTC qemu: generating initramfs
2025-06-24 07:21:59.693843291 +0000 UTC image configuration:
2025-06-24 07:21:59.693919973 +0000 UTC   contents:
2025-06-24 07:21:59.693931204 +0000 UTC     build repositories: [https://apk.cgr.dev/chainguard]
2025-06-24 07:21:59.693945646 +0000 UTC     runtime repositories: []
2025-06-24 07:21:59.693953721 +0000 UTC     keyring:      []
2025-06-24 07:21:59.693961442 +0000 UTC     packages:     [microvm-init]
2025-06-24 07:22:06.370088539 +0000 UTC installing wolfi-baselayout (20230201-r21)
2025-06-24 07:22:06.470048684 +0000 UTC installing ca-certificates-bundle (20241121-r42)
2025-06-24 07:22:06.471506067 +0000 UTC installing libgcc (15.1.0-r1)
2025-06-24 07:22:06.472654304 +0000 UTC installing glibc-locale-posix (2.41-r50)
2025-06-24 07:22:06.515362032 +0000 UTC installing glibc (2.41-r50)
2025-06-24 07:22:06.521897493 +0000 UTC installing ld-linux (2.41-r50)
2025-06-24 07:22:06.523337369 +0000 UTC installing gnutar (1.35-r4)
2025-06-24 07:22:06.531861 +0000 UTC installing libattr1 (2.5.2-r50)
2025-06-24 07:22:06.532820935 +0000 UTC installing attr (2.5.2-r50)
2025-06-24 07:22:06.534209327 +0000 UTC installing zlib (1.3.1-r50)
2025-06-24 07:22:06.535291373 +0000 UTC installing libzstd1 (1.5.7-r2)
2025-06-24 07:22:06.536976204 +0000 UTC installing xz (5.8.1-r1)
2025-06-24 07:22:06.541514464 +0000 UTC installing libcrypto3 (3.5.0-r3)
2025-06-24 07:22:06.54664781 +0000 UTC installing kmod (34.2-r40)
2025-06-24 07:22:06.548200493 +0000 UTC installing libbz2-1 (1.0.8-r18)
2025-06-24 07:22:06.54923111 +0000 UTC installing libelf (0.193-r1)
2025-06-24 07:22:06.551007109 +0000 UTC installing libmnl (1.0.5-r6)
2025-06-24 07:22:06.551985791 +0000 UTC installing libnftnl (1.2.9-r0)
2025-06-24 07:22:06.553594468 +0000 UTC installing iptables (1.8.11-r22)
2025-06-24 07:22:06.562462437 +0000 UTC installing iproute2 (6.15.0-r0)
2025-06-24 07:22:06.841037626 +0000 UTC installing libblkid (2.41-r42)
2025-06-24 07:22:06.842706826 +0000 UTC installing libmount (2.41-r42)
2025-06-24 07:22:06.844527636 +0000 UTC installing mount (2.41-r42)
2025-06-24 07:22:06.84531677 +0000 UTC installing ncurses-terminfo-base (6.5_p20241228-r3)
2025-06-24 07:22:06.85582508 +0000 UTC installing ncurses (6.5_p20241228-r3)
2025-06-24 07:22:06.858025879 +0000 UTC installing setarch (2.41-r42)
2025-06-24 07:22:06.859000561 +0000 UTC installing libuuid (2.41-r42)
2025-06-24 07:22:06.859760049 +0000 UTC installing libfdisk (2.41-r42)
2025-06-24 07:22:06.861571097 +0000 UTC installing sqlite-libs (3.50.1-r0)
2025-06-24 07:22:06.863079023 +0000 UTC installing util-linux (2.41-r42)
2025-06-24 07:22:06.867564537 +0000 UTC installing libsmartcols (2.41-r42)
2025-06-24 07:22:06.904770449 +0000 UTC installing util-linux-misc (2.41-r42)
2025-06-24 07:22:06.920255458 +0000 UTC installing libpcre2-8-0 (10.45-r2)
2025-06-24 07:22:06.921398275 +0000 UTC installing libsepol (3.8.1-r50)
2025-06-24 07:22:06.923202433 +0000 UTC installing libselinux (3.8.1-r50)
2025-06-24 07:22:06.935405189 +0000 UTC installing libxcrypt (4.4.38-r2)
2025-06-24 07:22:06.936852411 +0000 UTC installing libcrypt1 (2.41-r50)
2025-06-24 07:22:06.937906541 +0000 UTC installing linux-pam (1.7.1-r0)
2025-06-24 07:22:06.960330028 +0000 UTC installing openssh-keygen (10.0_p1-r1)
2025-06-24 07:22:06.961230276 +0000 UTC installing openssh-server-config (10.0_p1-r1)
2025-06-24 07:22:06.961825373 +0000 UTC installing openssh-server (10.0_p1-r1)
2025-06-24 07:22:06.964084971 +0000 UTC installing libstdc++ (15.1.0-r1)
2025-06-24 07:22:06.966609313 +0000 UTC installing inih (60-r0)
2025-06-24 07:22:06.967252383 +0000 UTC installing userspace-rcu (0.15.3-r0)
2025-06-24 07:22:06.970568129 +0000 UTC installing xfsprogs-core (6.14.0-r4)
2025-06-24 07:22:06.973090074 +0000 UTC installing xfsprogs (6.14.0-r4)
2025-06-24 07:22:06.973638569 +0000 UTC installing busybox (1.37.0-r46)
2025-06-24 07:22:06.974749636 +0000 UTC installing microvm-init (0.0.1-r11)
2025-06-24 07:22:08.198308211 +0000 UTC qemu: starting VM
2025-06-24 07:22:08.199586048 +0000 UTC qemu: waiting for SSH
2025-06-24 07:22:13.945246725 +0000 UTC qemu: meta-data=/dev/vda               isize=512    agcount=8, agsize=1638400 blks
2025-06-24 07:22:13.945357093 +0000 UTC qemu:          =                       sectsz=4096  attr=2, projid32bit=1
2025-06-24 07:22:13.945466627 +0000 UTC qemu:          =                       crc=1        finobt=1, sparse=1, rmapbt=1
2025-06-24 07:22:13.945482342 +0000 UTC qemu:          =                       reflink=1    bigtime=1 inobtcount=1 nrext64=1
2025-06-24 07:22:13.945492066 +0000 UTC qemu:          =                       exchange=0   metadir=0
2025-06-24 07:22:13.94549864 +0000 UTC qemu: data     =                       bsize=4096   blocks=13107200, imaxpct=25
2025-06-24 07:22:13.945506087 +0000 UTC qemu:          =                       sunit=0      swidth=0 blks
2025-06-24 07:22:13.945532176 +0000 UTC qemu: naming   =version 2              bsize=4096   ascii-ci=0, ftype=1, parent=0
2025-06-24 07:22:13.94554333 +0000 UTC qemu: log      =internal log           bsize=4096   blocks=16384, version=2
2025-06-24 07:22:13.945548487 +0000 UTC qemu:          =                       sectsz=4096  sunit=1 blks, lazy-count=1
2025-06-24 07:22:13.945557773 +0000 UTC qemu: realtime =none                   extsz=4096   blocks=0, rtextents=0
2025-06-24 07:22:13.945563725 +0000 UTC qemu:          =                       rgcount=0    rgsize=0 extents
2025-06-24 07:22:13.946411903 +0000 UTC qemu: Discarding blocks...Done.
2025-06-24 07:22:18.989353143 +0000 UTC qemu: ssh-keygen: generating new host keys: RSA ECDSA ED25519 
2025-06-24 07:22:18.997094578 +0000 UTC qemu: Server listening on 0.0.0.0 port 2223.
2025-06-24 07:22:18.997151118 +0000 UTC qemu: Server listening on 0.0.0.0 port 22.
2025-06-24 07:22:19.196575533 +0000 UTC qemu: Connection closed by 10.0.2.2 port 50304
2025-06-24 07:22:19.224115741 +0000 UTC qemu: VM started successfully, SSH server is up
2025-06-24 07:22:19.224328018 +0000 UTC qemu: Connection closed by 10.0.2.2 port 45390
2025-06-24 07:22:19.249162332 +0000 UTC qemu: Accepted publickey for root from 10.0.2.2 port 45394 ssh2: ECDSA SHA256:gbAp4H4FOpMW2DVGdU4Epc1vjIeFieDZQhZe/hOhpnY
2025-06-24 07:22:19.25634081 +0000 UTC qemu: Connection closed by 10.0.2.2 port 45394
2025-06-24 07:22:19.279909551 +0000 UTC qemu: Accepted publickey for root from 10.0.2.2 port 45404 ssh2: ECDSA SHA256:gbAp4H4FOpMW2DVGdU4Epc1vjIeFieDZQhZe/hOhpnY
2025-06-24 07:22:19.30987244 +0000 UTC qemu: Accepted publickey for root from 10.0.2.2 port 38806 ssh2: ECDSA SHA256:gbAp4H4FOpMW2DVGdU4Epc1vjIeFieDZQhZe/hOhpnY
2025-06-24 07:22:19.315454654 +0000 UTC qemu: setting up melange cachedir: ./melange-cache/
2025-06-24 07:22:19.328874216 +0000 UTC qemu: setting up local workspace
2025-06-24 07:22:19.428333339 +0000 UTC running step "fetch"
2025-06-24 07:22:19.445611282 +0000 UTC --2025-06-24 07:22:19--  https://www.ex-parrot.com/~pdw/iftop/download/iftop-0.17.tar.gz
2025-06-24 07:22:19.562175318 +0000 UTC Resolving www.ex-parrot.com... 46.235.227.5, 2a00:1098:82::e:1
2025-06-24 07:22:19.67547194 +0000 UTC Connecting to www.ex-parrot.com|46.235.227.5|:443... connected.
2025-06-24 07:22:19.911831964 +0000 UTC HTTP request sent, awaiting response... 301 Moved Permanently
2025-06-24 07:22:19.911874143 +0000 UTC Location: https://pdw.ex-parrot.com/iftop/download/iftop-0.17.tar.gz [following]
2025-06-24 07:22:19.911880193 +0000 UTC --2025-06-24 07:22:19--  https://pdw.ex-parrot.com/iftop/download/iftop-0.17.tar.gz
2025-06-24 07:22:19.953087754 +0000 UTC Resolving pdw.ex-parrot.com... 46.235.227.5, 2a00:1098:82::e:1
2025-06-24 07:22:20.04817863 +0000 UTC Connecting to pdw.ex-parrot.com|46.235.227.5|:443... connected.
2025-06-24 07:22:20.276507896 +0000 UTC HTTP request sent, awaiting response... 200 OK
2025-06-24 07:22:20.27669804 +0000 UTC Length: 160381 (157K) [application/gzip]
2025-06-24 07:22:20.276736564 +0000 UTC Saving to: 'iftop-0.17.tar.gz'
2025-06-24 07:22:20.276743556 +0000 UTC 
2025-06-24 07:22:20.46567555 +0000 UTC      0K .......... .......... .......... .......... .......... 31%  264K 0s
2025-06-24 07:22:20.556823318 +0000 UTC     50K .......... .......... .......... .......... .......... 63%  550K 0s
2025-06-24 07:22:20.560275405 +0000 UTC    100K .......... .......... .......... .......... .......... 95% 12.8M 0s
2025-06-24 07:22:20.560371178 +0000 UTC    150K ......                                                100%  296M=0.3s
2025-06-24 07:22:20.560407948 +0000 UTC 
2025-06-24 07:22:20.56047042 +0000 UTC 2025-06-24 07:22:20 (552 KB/s) - 'iftop-0.17.tar.gz' saved [160381/160381]
2025-06-24 07:22:20.560486193 +0000 UTC 
2025-06-24 07:22:20.561765182 +0000 UTC fetch: Expected sha256: d032547c708307159ff5fd0df23ebd3cfa7799c31536fa0aea1820318a8e0eac
2025-06-24 07:22:20.57415574 +0000 UTC running step "autoconf/configure"
2025-06-24 07:22:20.691700872 +0000 UTC checking build system type... x86_64-pc-linux-gnu
2025-06-24 07:22:20.704680477 +0000 UTC checking host system type... x86_64-pc-linux-gnu
2025-06-24 07:22:20.717198345 +0000 UTC checking target system type... x86_64-pc-linux-gnu
2025-06-24 07:22:20.755455003 +0000 UTC checking for a BSD-compatible install... /usr/bin/install -c
2025-06-24 07:22:21.760554277 +0000 UTC checking whether build environment is sane... yes
2025-06-24 07:22:21.766881666 +0000 UTC checking for gawk... no
2025-06-24 07:22:21.76700584 +0000 UTC checking for mawk... no
2025-06-24 07:22:21.767044195 +0000 UTC checking for nawk... no
2025-06-24 07:22:21.767053145 +0000 UTC checking for awk... awk
2025-06-24 07:22:21.7755035 +0000 UTC checking whether make sets ${MAKE}... yes
2025-06-24 07:22:21.782352811 +0000 UTC checking for x86_64-pc-linux-gnu-gcc... x86_64-pc-linux-gnu-gcc
2025-06-24 07:22:21.8499342 +0000 UTC checking for C compiler default output... a.out
2025-06-24 07:22:21.85351468 +0000 UTC checking whether the C compiler works... yes
2025-06-24 07:22:21.855657477 +0000 UTC checking whether we are cross compiling... no
2025-06-24 07:22:21.899923082 +0000 UTC checking for suffix of executables... 
2025-06-24 07:22:21.940315152 +0000 UTC checking for suffix of object files... o
2025-06-24 07:22:21.971416863 +0000 UTC checking whether we are using the GNU C compiler... yes
2025-06-24 07:22:22.005214685 +0000 UTC checking whether x86_64-pc-linux-gnu-gcc accepts -g... yes
2025-06-24 07:22:22.059711158 +0000 UTC checking for x86_64-pc-linux-gnu-gcc option to accept ANSI C... none needed
2025-06-24 07:22:22.089792836 +0000 UTC checking for style of include used by make... GNU
2025-06-24 07:22:22.118620707 +0000 UTC checking dependency style of x86_64-pc-linux-gnu-gcc... gcc3
2025-06-24 07:22:22.161479692 +0000 UTC checking how to run the C preprocessor... x86_64-pc-linux-gnu-gcc -E
2025-06-24 07:22:22.209658018 +0000 UTC checking for egrep... grep -E
2025-06-24 07:22:22.340920246 +0000 UTC checking for ANSI C header files... yes
2025-06-24 07:22:22.381481772 +0000 UTC checking for sys/types.h... yes
2025-06-24 07:22:22.420674697 +0000 UTC checking for sys/stat.h... yes
2025-06-24 07:22:22.459700136 +0000 UTC checking for stdlib.h... yes
2025-06-24 07:22:22.51005025 +0000 UTC checking for string.h... yes
2025-06-24 07:22:22.558216479 +0000 UTC checking for memory.h... yes
2025-06-24 07:22:22.603553004 +0000 UTC checking for strings.h... yes
2025-06-24 07:22:22.646255727 +0000 UTC checking for inttypes.h... yes
2025-06-24 07:22:22.68926612 +0000 UTC checking for stdint.h... yes
2025-06-24 07:22:22.737303096 +0000 UTC checking for unistd.h... yes
2025-06-24 07:22:22.790254254 +0000 UTC checking sys/ioctl.h usability... yes
2025-06-24 07:22:22.810838181 +0000 UTC checking sys/ioctl.h presence... yes
2025-06-24 07:22:22.811586819 +0000 UTC checking for sys/ioctl.h... yes
2025-06-24 07:22:22.862137074 +0000 UTC checking sys/time.h usability... yes
2025-06-24 07:22:22.880574776 +0000 UTC checking sys/time.h presence... yes
2025-06-24 07:22:22.881247387 +0000 UTC checking for sys/time.h... yes
2025-06-24 07:22:22.924966776 +0000 UTC checking sys/sockio.h usability... no
2025-06-24 07:22:22.942801504 +0000 UTC checking sys/sockio.h presence... no
2025-06-24 07:22:22.943591505 +0000 UTC checking for sys/sockio.h... no
2025-06-24 07:22:22.94612606 +0000 UTC checking for unistd.h... (cached) yes
2025-06-24 07:22:22.977186816 +0000 UTC checking for an ANSI C-conforming const... yes
2025-06-24 07:22:23.024713262 +0000 UTC checking for size_t... yes
2025-06-24 07:22:23.054489783 +0000 UTC checking whether time.h and sys/time.h may both be included... yes
2025-06-24 07:22:23.098401161 +0000 UTC checking sys/dlpi.h usability... no
2025-06-24 07:22:23.117878539 +0000 UTC checking sys/dlpi.h presence... no
2025-06-24 07:22:23.11792772 +0000 UTC checking for sys/dlpi.h... no
2025-06-24 07:22:23.163118042 +0000 UTC checking for regcomp... yes
2025-06-24 07:22:23.212149048 +0000 UTC checking for select... yes
2025-06-24 07:22:23.267302097 +0000 UTC checking for strdup... yes
2025-06-24 07:22:23.319570399 +0000 UTC checking for strerror... yes
2025-06-24 07:22:23.372198476 +0000 UTC checking for strspn... yes
2025-06-24 07:22:23.419609231 +0000 UTC checking for library containing socket... none required
2025-06-24 07:22:23.52642295 +0000 UTC checking for library containing log... -lm
2025-06-24 07:22:24.612926192 +0000 UTC checking for gethostbyname... yes
2025-06-24 07:22:24.660165694 +0000 UTC checking for library containing inet_aton... none required
2025-06-24 07:22:24.704850373 +0000 UTC checking for library containing inet_pton... none required
2025-06-24 07:22:24.753214035 +0000 UTC checking for inet_aton... yes
2025-06-24 07:22:24.805854306 +0000 UTC checking for inet_pton... yes
2025-06-24 07:22:24.865308729 +0000 UTC checking size of u_int8_t... 1
2025-06-24 07:22:24.920814948 +0000 UTC checking size of u_int16_t... 2
2025-06-24 07:22:24.995120051 +0000 UTC checking size of u_int32_t... 4
2025-06-24 07:22:25.047696132 +0000 UTC checking for library containing getnameinfo... none required
2025-06-24 07:22:25.09354061 +0000 UTC checking for library containing gethostbyaddr_r... none required
2025-06-24 07:22:25.1986587 +0000 UTC checking how to call gethostbyaddr_r... 8 args, int return
2025-06-24 07:22:25.281747152 +0000 UTC checking gethostbyaddr_r usability... yes
2025-06-24 07:22:25.311119486 +0000 UTC checking where to find pcap.h... /include
2025-06-24 07:22:25.356547562 +0000 UTC checking for pcap_open_live in -lpcap... yes
2025-06-24 07:22:25.408805351 +0000 UTC checking pcap.h usability... yes
2025-06-24 07:22:25.436088935 +0000 UTC checking pcap.h presence... yes
2025-06-24 07:22:25.436840682 +0000 UTC checking for pcap.h... yes
2025-06-24 07:22:25.535608075 +0000 UTC configure: error: Curses! Foiled again!
2025-06-24 07:22:25.535624244 +0000 UTC checking for a curses library containing mvchgat... none found
2025-06-24 07:22:25.535665853 +0000 UTC   (Can't find a curses library supporting mvchgat.)
2025-06-24 07:22:25.535674294 +0000 UTC   Consider installing ncurses.
2025-06-24 07:22:25.577598745 +0000 UTC Failed wait for session running: "/bin/sh -c 'set -e 
[ -d '\''/home/build'\'' ] || mkdir -p '\''/home/build'\''
cd '\''/home/build'\''
cd .
if [[ ! -f ./configure && -f ./configure.ac ]]; then
	autoreconf -vfi
fi
./configure \
	--host=x86_64-pc-linux-gnu \
	--build=x86_64-pc-linux-gnu \
	--prefix=/usr \
	--sysconfdir=/etc \
	--libdir=/usr/lib \
	--mandir=/usr/share/man \
	--infodir=/usr/share/info \
	--localstatedir=/var \
	--sbindir=/usr/bin

exit 0'": Process exited with status 1
2025-06-24 07:22:25.577683173 +0000 UTC qemu: sending shutdown signal
2025-06-24 07:22:25.898859695 +0000 UTC failed to build package: unable to run package iftop pipeline: unable to run pipeline: unable to run pipeline: Process exited with status 1
2025-06-24 07:22:25.944386436 +0000 UTC failed to build: command melange failed: exit status 1
2025-06-24 07:22:25.944559717 +0000 UTC build failed: command melange failed: exit status 1
```

--- a/iftop.yaml
+++ b/iftop.yaml
@@ -1,7 +1,7 @@
 package:
   name: iftop
   version: 0.17
-  epoch: 41
+  epoch: 42
   description: A tool to display bandwidth usage on an interface
   copyright:
     - license: GPL-2.0-or-later
@@ -21,6 +21,7 @@ environment:
       - gcc-14-default
       - libpcap-dev
       - ncurses-dev
+      - ncurses-static
   environment:
     CFLAGS: "-Wno-unused-function -Wno-implicit-function-declaration"
 

